types/node/http.d.ts

```diff
@@ -156,7 +156,8 @@ declare module 'http' {
 
     // https://github.com/nodejs/node/blob/master/lib/_http_outgoing.js
     class OutgoingMessage extends stream.Writable {
-        upgrading: boolean;
+        readonly req: IncomingMessage;
+
```

Comment:
May I ask why this change has been introduced? I see above that you added the same kind of thing here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/52775/files#r640537682

I ask the question because I'm working on a web framework with my team and this is a breaking change of us.